### PR TITLE
Adds Feature: Pascalify

### PR DESCRIPTION
**Pull request by Nekoniri**
(ThanksForSendingAPullRequest! PleaseMakeSureYouClickTheLinkAboveToViewTheContributionGuidelines, ThenFillOutTheBlanksBelow.)

WhatDoesThisImplement/Fix? ExplainYourChanges.
---------------------------------------------------
ItAddsACommandThatAllowsUsersToConvertTheirMessageToPascalCase

DoesThisCloseAnyCurrentlyOpenIssues?
-------------------------------------------
NoButItHelpsResolveOurDisturbingLackOfPascalCase

AnyRelevantLogs, ErrorOutput, Etc?
-------------------------------------
(IfIt’sLong, PleasePasteTo https://hastebin.com/ AndInsertTheLinkHere.)

AnyOtherComments?
-------------------
NoComments

WhereHasThisBeenTested?
---------------------------
NoTestingHasBeenDoneYet

### CODE REVIEW

**Comment by Haxified** [comment]
jesus

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
         "discord.js": "discordjs/discord.js",
         "dotenv": "^8.2.0",
         "ms": "^2.1.2",
+        "pascal-case": "^2.0.1",
```
Comment (Haxified) [inline package.json]: do we really need a new dep?

File: src/commands/pascalify.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { Message } from 'discord.js';
+
+import { Command } from '../utils/commandHandler';
+
+import PascalCase from 'pascal-case';
+
+export const command = new Command({
+    description: 'Converts user message to PascalCase',
+    aliases: ['PascalCase', 'Pascalify', 'pascalify', 'Robinify'],
+    command: async (message: Message): Promise<Message> => message.channel.send(PascalCase(message.content)),
```
Comment (Haxified) [inline src/commands/pascalify.ts]: this.. dosent return a message

Comment (Nekoniri) [reply src/commands/pascalify.ts]: How? message.channel.send has Promise<Message> as return type?

File: package.json (hunk 1 above)
Comment (Nekoniri) [reply package.json]: Don't merge for now, I'll rewrite it proper later then :)

File: src/commands/pascalify.ts (hunk 2 above)
Comment (Haxified) [reply src/commands/pascalify.ts]: ahh nvm, i thought there were braces haha

File: package.json (hunk 1 above)
Comment (Nekoniri) [reply package.json]: It's gone :)

**Comment by resynth1943** [comment]
WHAT_ABOUT_SCREAMING_SNAKE_CASE_IT_IS_SO_GOOD_MAYBE_YOU_SHOULD_USE_PYTHON_INSTEAD_JUST_FOR_SCREAMING_SNAKE_CASE

**Pull request merged**